index.ts

```diff
@@ -53,4 +53,6 @@ const app = createExpressServer({
   defaultErrorHandler: false,
 });
 
+app.set('trust proxy', 1);
```

Comment:
Also this is ugly as well, it feels weird adding this app.set here. I don't think there's a way with our routing-controllers package to have the app.set config set, but maybe there's other ideas out there